README.md

```diff
@@ -33,3 +34,13 @@ async def testDaikin():
 
 asyncio.run(testDaikin())
 ```
+
+## Firmware Version 2.8.0 Support
+
+Firmware version 2.8.0 uses a different API structure compared to earlier firmware versions. The library now automatically detects the firmware version and uses the appropriate communication method. Confirmed working with:
+
+* FTKM20YVMA with firmware version 2.8.0
+* FTXM46WVMA with firmware version 2.8.0
+* FTXV80WVMA with firmware version 2.8.0
```

Comment:
```suggestion
* FTXV80WVMA with firmware version 2.8.0
* FTXA25C2V1BW with firmware version 2.8.0
* FTXA50C2V1BW with firmware version 2.8.0
```

According to: https://github.com/home-assistant/core/issues/125584#issuecomment-2854839579